pages/1.12/cli/experiments/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: layout.pug
+navigationTitle:  Experiments
+title: Experiments
+menuWeight: 1
+excerpt: What experimental features are currently available and how to enable them
+
+enterprise: false
+---
+
+Experimental features are features that are currently in development and
```

Comment:
I would change this paragraph for: The DC/OS CLI offers experimental features, those are in development and may still contain bugs.